scripts/list_tiles.py

```diff
@@ -72,8 +72,9 @@ def Col(self, x):
 
     def Digits(self, number):
         digits = 1 if (number < 0) else 0
+        number = number
         while number:
-            number /= 10
+            number = int(number / 10)
```

Comment:
I needed that in another context. And realized it's not python3 ported yet. python2 would've output an int by default, python3 outputs a float by default if it needs to.